src/OSPSuite.Presentation/Services/ParameterIdentifications/PredictedVsObservedChartService.cs

```diff
@@ -106,16 +111,24 @@ public void SetXAxisDimension(IEnumerable<DataColumn> observationColumns, Parame
          xAxis.Scaling = chart.AxisBy(AxisTypes.Y).Scaling;
          xAxis.UnitName = chart.AxisBy(AxisTypes.Y).UnitName;
 
-         chart.UpdateAxesVisibility();
+         UpdateAxesVisibility(chart);
       }
 
       public IReadOnlyList<DataRepository> AddIdentityCurves(IEnumerable<DataColumn> observationColumns, ParameterIdentificationPredictedVsObservedChart chart)
       {
          var identityCurves = addIdentityCurves(observationColumns, chart).ToList();
-         chart.UpdateAxesVisibility();
+         UpdateAxesVisibility(chart);
          return identityCurves;
       }
 
+      public void UpdateAxesVisibility(ParameterIdentificationPredictedVsObservedChart chart)
```

Comment:
Is there a reason why you would move this concern outside of the Chart when you already have a specific chart object?